src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -684,6 +729,19 @@ func (a *StorageMinerActorCode_I) ProveCommitSector(rt Runtime, info sector.Sect
 	return rt.SuccessReturn()
 }
 
+func (a *StorageMinerActorCode_I) _ensurePledgeCollateralSatisfied(rt Runtime) {
+	emptyParams := make([]actor.MethodParam, 0)
+	ret := rt.SendCatchingErrors(&msg.InvocInput_I{
+		To_:     addr.StoragePowerActorAddr,
+		Method_: power.EnsurePledgeCollateralSatisfied,
+		Params_: emptyParams,
+	})
+
+	if ret.ExitCode() == exitcode.InsufficientPledgeCollateral {
+		rt.Abort("sma._onSuccessfulPoSt: insufficient pledge collateral.")
+	}
```

Comment:
What's going on here, it's catching the error only to re-Abort? What other error are you expecting that you don't want to cause an abort?

Reply:
See also #644 (in particular https://github.com/filecoin-project/specs/pull/644/files#r351364839) where I had to change this call in order to update for the new error handling API -- ideally we should resolve merge conflicts w/that PR as soon as possible.